protocol/contracts/beanstalk/silo/SiloFacet/SiloFacet.sol

```diff
@@ -299,6 +299,10 @@ contract SiloFacet is TokenSilo {
         return _plant(msg.sender, token);
     }
 
+    function plantForAccount(address account) external payable returns (uint256 beans) {
+        return _plant(account);
+    }
```

Comment:
I suppose the idea is to expose this so that anyone can plant for anyone else is for Tractor purposes? If you plant in the morning then can you still plant later within the same season? Just trying to think if there's something malicious you could do if you planted for someone else (like slip in a plant tx right before their <whatever> tx if you see someone is trying to do something with another tx that's visible in the mempool and planting for them could mess them up somehow)